src/integrations/admin/workouts-integration.php

```diff
@@ -171,102 +129,16 @@ public function render_target() {
 		echo '<div id="wpseo-workouts-container-free"></div>';
 	}
 
-	/**
-	 * Retrieves the public indexable sub types.
-	 *
-	 * @return array The sub types.
-	 */
-	protected function get_public_sub_types() {
-		$object_sub_types = \array_values(
-			\array_merge(
-				$this->post_type_helper->get_public_post_types(),
-				\get_taxonomies( [ 'public' => true ] )
-			)
-		);
-
-		$excluded_post_types = \apply_filters( 'wpseo_indexable_excluded_post_types', [ 'attachment' ] );
-		$object_sub_types    = \array_diff( $object_sub_types, $excluded_post_types );
-		return $object_sub_types;
-	}
-
 	/**
 	 * Gets the workouts option and extends it with indexable data.
```

Comment:
```suggestion
	 * Gets the workouts option.
```
or
```suggestion
	 * Gets the workouts option and applies a filter for add-ons to add their workouts options.
```

Reply:
I'd use the first, simpler one. Good idea.